site/app/views/calendar/CalendarView.php

```diff
@@ -59,7 +59,8 @@ public function showCalendar(CalendarInfo $info, bool $in_course = false): strin
             "empty_message" => $info->getEmptyMessage(),
             "in_course" => $in_course,
             "is_instructor" => $this->core->getUser()->getGroup() === User::GROUP_INSTRUCTOR,
-            "colors" => $info->getColors()
+            "colors" => $info->getColors(),
+            "cookies" => $_COOKIE
```

Comment:
Probably better to just pass what's relevant for cookies rather than every cookie. So just $_COOKIE['view'] assuming you don't need anything else.

Reply:
Thank you! This makes it a lot cleaner in the twig too 👍